test/testData/generate/getHooks.ts

```diff
@@ -0,0 +1,62 @@
+import { formatEther, parseEther } from 'viem';
+import config from '../../../config';
+import { chainIdToChain } from '../../../modules/network/chain-id-to-chain';
+import { fetchHookData } from '../../../modules/sources/contracts/v3/fetch-hook-data';
+import { getViemClient } from '../../../modules/sources/viem-client';
+import { HookData } from '../../../prisma/prisma-types';
+import { PoolBase } from '../types';
+
+export async function getHooks(pools: PoolBase[], chainId: number, blockNumber: bigint) {
```

Comment:
Might be better to name as `getPoolsWithHooks`?

Reply:
You're right, `getHooks` does not represent the function behavior.
Initial implementation was meant to get hooks, but I ended up adapting it and forgot to rename 😅 
I think `enrichPoolsWithHookData` is even more descriptive.
I'll go with that - let me know what you think 😉